rs/cli/src/cli.rs

```diff
@@ -1,8 +1,10 @@
+use std::path::PathBuf;
+
 use clap::{Parser, Subcommand};
 use clap_num::maybe_hex;
 use ic_base_types::PrincipalId;
 use ic_management_types::{Artifact, Network};
-use log::error;
+use slog::{error, Logger};
```

Comment:
Ideally, switch from log -> slog would have gone into a separate PR. Having it together with the other change makes it harder to review.

Reply:
The integrated functionality requires serilog so I had to add it all in one PR